datashader/data_libraries/dask.py

```diff
@@ -15,6 +18,16 @@
 
 __all__ = ()
 
+# Warn if query planning is installed
+query_planning = dask.config.get("dataframe.query-planning")
+dask_expr = find_spec("dask_expr")
+
+if query_planning or (query_planning is None and dask_expr):
+    msg = """\
+Dask query planning is enabled. This does not currently work with datashader.
```

Comment:
dask_expr does not currently work with datashader so I have chosen to emit a warning disabling.